lib/utils/filter_samples.py

```diff
@@ -130,9 +137,24 @@ def _format_and_validate_filters(self, parsed_filters):
                 message = "Unable to resolve metadata fields or prefix metadata fields: " + \
                           ", ".join(list(key_set))
                 raise ValueError(message)
+
+        # check if there are any bad uncontrolled fields
+        if len(uc_filters):
+            fields = MetadataManager(self.re_api_url).get_sampleset_meta(samples, token)['results']
+            uc_fields = {f['field'] for f in uc_filters}
+            missing_fields = uc_fields.difference(set(fields))
+            if len(missing_fields):
+                message = "Unable to resolve uncontrolled custom metadata fields: " + \
+                    ", ".join(list(missing_fields))
+                raise ValueError(message)
+
+
         formatted_filters = []
         for idx, parsed_filter in enumerate(parsed_filters):
-            stat_meta = static_metadata.get(parsed_filter.get('field'))
+            if parsed_filter.get('field', '').startswith('custom:'):
+                stat_meta = {'type': 'any'}  # no validation for uncontrolled fields
+            else:
+                stat_meta = static_metadata.get(parsed_filter.get('field'))
             formatted_filters.append(
                 field_value_formatting(parsed_filter, stat_meta, idx)
             )
```

Comment:
another minor clarity thing
```suggestion
            filter_field = parsed_filter.get('field', '')
            if filter_field.startswith('custom:'):
                stat_meta = {'type': 'any'}  # no validation for uncontrolled fields
            else:
                stat_meta = static_metadata.get(filter_field)
            formatted_filters.append(
                field_value_formatting(parsed_filter, stat_meta, idx)
            )
```